samples/client/petstore/go/go-petstore/user_api.go

```diff
@@ -400,9 +400,9 @@ func (a *UserApiService) LoginUser(username string, password string) (string,  *
 
 /* UserApiService Logs out current logged in user session
  
-
+ * @param ctx context.Context for authentication, logging, tracing, etc.
  @return */
-func (a *UserApiService) LogoutUser() ( *http.Response, error) {
+func (a *UserApiService) LogoutUser(ctx context.Context, ) ( *http.Response, error) {
```

Comment:
@antihax I think there's a way to remove the trailing comma when the endpoint does not require any parameter. Let me know if you need help on this.